tests/TokenizerTest.php

```diff
@@ -24,6 +26,1492 @@
+        'PCTFREE',
+        'PRIOR',
+        'PUBLIC',
+        'RAW',
+        'RENAME',
+        'RESOURCE',
+        'REVOKE',
+        'ROW',
+        'ROWID',
+        'ROWNUM',
+        'ROWS',
+        'SELECT',
+        'SESSION',
+        'SET',
+        'SHARE',
+        'SIZE',
+        'SMALLINT',
+        'START',
+        'SUCCESSFUL',
+        'SYNONYM',
+        'SYSDATE',
+        'TABLE',
+        'THEN',
+        'TO',
+        'TRIGGER',
+        'UID',
+        'UNION',
+        'UNIQUE',
+        'UPDATE',
+        'USER',
+        'VALIDATE',
+        'VALUES',
+        'VARCHAR',
+        'VARCHAR2',
+        'VIEW',
+        'WHENEVER',
+        'WHERE',
+        'WITH',
+    ];
+
+    /**
+     * Based on https://www.postgresql.org/docs/16/sql-keywords-appendix.html list.
+     *
+     * All these keywords must be quoted.
+     */
+    private const KEYWORDS_RESERVED_POSTGRESQL = [
+        'ALL',
+        'ANALYSE',
+        'ANALYZE',
+        'AND',
+        'ANY',
+        'ARRAY',
+        'AS',
+        'ASC',
+        'ASYMMETRIC',
+        'AUTHORIZATION',
+        'BINARY',
+        'BOTH',
+        'CASE',
+        'CAST',
+        'CHECK',
+        'COLLATE',
+        'COLLATION',
+        'COLUMN',
+        'CONCURRENTLY',
+        'CONSTRAINT',
+        'CREATE',
+        'CROSS',
+        'CURRENT_CATALOG',
+        'CURRENT_DATE',
+        'CURRENT_ROLE',
+        'CURRENT_SCHEMA',
+        'CURRENT_TIME',
+        'CURRENT_TIMESTAMP',
+        'CURRENT_USER',
+        'DEFAULT',
+        'DEFERRABLE',
+        'DESC',
+        'DISTINCT',
+        'DO',
+        'ELSE',
+        'END',
+        'EXCEPT',
+        'FALSE',
+        'FETCH',
+        'FOR',
+        'FOREIGN',
+        'FREEZE',
+        'FROM',
+        'FULL',
+        'GRANT',
+        'GROUP',
+        'HAVING',
+        'ILIKE',
+        'IN',
+        'INITIALLY',
+        'INNER',
+        'INTERSECT',
+        'INTO',
+        'IS',
+        'ISNULL',
+        'JOIN',
+        'LATERAL',
+        'LEADING',
+        'LEFT',
+        'LIKE',
+        'LIMIT',
+        'LOCALTIME',
+        'LOCALTIMESTAMP',
+        'NATURAL',
+        'NOT',
+        'NOTNULL',
+        'NULL',
+        'OFFSET',
+        'ON',
+        'ONLY',
+        'OR',
+        'ORDER',
+        'OUTER',
+        'OVERLAPS',
+        'PLACING',
+        'PRIMARY',
+        'REFERENCES',
+        'RETURNING',
+        'RIGHT',
+        'SELECT',
+        'SESSION_USER',
+        'SIMILAR',
+        'SOME',
+        'SYMMETRIC',
+        'SYSTEM_USER',
+        'TABLE',
+        'TABLESAMPLE',
+        'THEN',
+        'TO',
+        'TRAILING',
+        'TRUE',
+        'UNION',
+        'UNIQUE',
+        'USER',
+        'USING',
+        'VARIADIC',
+        'VERBOSE',
+        'WHEN',
+        'WHERE',
+        'WINDOW',
+        'WITH',
+    ];
+
+    /**
+     * Based on https://www.postgresql.org/docs/16/sql-keywords-appendix.html list.
```

Comment:
That link looks like a copy/paste

Reply:
The PostgreSQL link is intended there as the SQL:2023 spec is non-public/paid - I find PostgreSQL as quite reputable source and then I cross verified it againts https://en.wikipedia.org/wiki/List_of_SQL_reserved_words. Is the actual comment fine or should I update it in any way?

Reply:
It's fine, just wanted to make sure.